en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md

```diff
@@ -122,6 +122,9 @@ After applying the above change, you can start the Micro Integrator with the fol
 
 Once you have successfully set up the [minimum deployment](#step-1-set-up-the-minimum-deployment), you need to set up the log processing add-on to process logs. To achieve this, you can use Grafana Loki-based logging stack.
 
+!!! note
+    You do not need the minimum deployment to set this up and observe the traces through Jaeger. 
+
```

Comment:
This should be applied under Step 3 - Optionally, integrate the Message Tracing add-on.

Reply:
Fixed.